.nvmrc

```diff
@@ -1 +1 @@
-v9.5.0
+v9.11.1
```

Comment:
I guess we could go 10.x now that it's out

Reply:
I've tried them all, we have restrictions from dependencies to run.

Actually:
	* examples/server-side-rendering: node8